tests/puppeteer/campaigns/functional/BO/customers/customers/01_filter&QuickEditCustomers.js

```diff
@@ -0,0 +1,313 @@
+// Using chai
+const {expect} = require('chai');
+const helper = require('../../../../utils/helpers');
```

Comment:
It's maybe time to add an alias (like for webpack) to do `require('~/utils/helpers')` for example? 
(don't be scared, can be added in a new PR)

Reply:
Thanks for suggestion, I will add that in a new PR.
@PierreRambaud , Can you approved if its all good for you ?

Reply:
I recommend you to use https://www.npmjs.com/package/module-alias instead which is more maintained and use by a lot of good js libraries :)